data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml

```diff
@@ -0,0 +1,57 @@
+  Free energy data for each alchemical lambda is stored in the md.log file, and found more concisely in pre-scraped pkl dataframes.
+  The dataset comprises several projects, each having a RUN*/CLONE*/result* directory structure.
+  Each PROJ represents a different dataset of small molecules. Each RUN represents a different small molecule,
+  each CLONE is a unique MD run differing in initial atomic velocities, and result is a fragment of the contiguous simulation.
+  For the case of ligand-only "L" simulations, each xtc represents 10ns of sampling, while each complex "RL" xtc trajectory is only 1ns.
+  More information about the simulation set-up can be found [here](https://github.com/FoldingAtHome/covid-moonshot/blob/master/folding-at-home_submissions/COVID%20Moonshot%20051220%20Release%20Announcement.pdf).
+  In order to find particular PROJ/RUN of interest, see the results and organization dataframes.
+  
+  *Entire dataset:* To get the raw trajectory files (XTCs) of the whole dataset (~10+ TB/project), you can use the [AWS CLI](https://aws.amazon.com/cli/):
+   ```bash
+   aws s3 --no-sign-request sync s3://fah-public-data-covid19-absolute-free-energy/SVR51748107/PROJ14721 .
+   aws s3 --no-sign-request sync s3://fah-public-data-covid19-absolute-free-energy/SVR51748107/PROJ14722/RUN2 .
+   aws s3 --no-sign-request sync s3://fah-public-data-covid19-absolute-free-energy/SVR51748107/PROJ14725/RUN2/CLONE1 .
+   aws s3 --no-sign-request sync s3://fah-public-data-covid19-absolute-free-energy/SVR51748107/PROJ14727/RUN2/CLONE1/results0 .
+   aws s3 --no-sign-request sync s3://fah-public-data-covid19-absolute-free-energy/SVR51748107/PROJ14728/RUN2/CLONE1/results0/traj_comp.xtc .
+   aws s3 --no-sign-request sync s3://fah-public-data-covid19-absolute-free-energy/SVR51748107/PROJ14752 .
+   ```
+   *Build directories:* To pull the build directory for a given project (including structure, topology, MD parameters for a given project/run, access via [AWS CLI](https://aws.amazon.com/cli):
+    ```bash
+    aws s3 --no-sign-request sync s3://fah-public-data-covid19-absolute-free-energy/build/p14721 .
+    aws s3 --no-sign-request sync s3://fah-public-data-covid19-absolute-free-energy/build/p14722/RUN0 .
+    aws s3 --no-sign-request sync s3://fah-public-data-covid19-absolute-free-energy/build/p14725/RUN10/npt.gro .
+    ```
+  *Organization dataframe:* In order to determine systems of interest, you can parse [this dataframe](https://fah-public-data-covid19-absolute-free-energy/free_energy_data/organization.pkl) which contains information on which ligands are present in each project/run. Half of the projects contain trajectories for ligands alone in solution, while the other half contain trajectories for ligands docked to the main protease.  More data is being added to this repository every day, so if a project/run is not available in the AWS bucket, check back later.
```

Comment:
This might go first, then the build directories, then results, then the raw datasets.